airflow/www/extensions/init_jinja_globals.py

```diff
@@ -44,6 +44,7 @@ def init_jinja_globals(app):
 
     def prepare_jinja_globals():
         extra_globals = {
+            'app_name': conf.get(section="webserver", key="site_title", fallback="Airflow"),
```

Comment:
We could do this without an extra variable, using `{{ appbuilder.app_name }}` in the templates (which will already be set up for us)

Reply:
Hi @ashb just wanted to double check: I've removed `app_name` in Line:47 and have replaced `app_name` in the *.html files with `{{ appbuilder.app_name }}`. Just wanted to make sure I understood correctly.